src/WebJobs.Script/Diagnostics/FileLogger.cs

```diff
@@ -53,12 +51,20 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
                 return;
             }
 
+            TraceLevel traceLevel = GetTraceLevel(logLevel);
+            TraceEvent traceEvent = new TraceEvent(traceLevel, formattedMessage, _categoryName, exception);
+            string functionName = GetFunctionName();
```

Comment:
I'm pretty sure I was catching that and returning if it was null at some point. I may have undone that in my refactoring. Let me add a test and fix this.